stack/rest/src/test/java/org/apache/usergrid/rest/management/RegistrationIT.java

```diff
@@ -17,212 +17,238 @@
-        String body = ( ( MimeMultipart ) msg.getContent() ).getBodyPart( 0 ).getContent().toString();
-        String token = StringUtils.substringAfterLast( body, "token=" );
-        // TODO better token extraction
-        // this is going to get the wrong string if the first part is not
-        // text/plain and the url isn't the last character in the email
-        return token;
-    }
-
-
-
-    @SuppressWarnings({ "unchecked", "rawtypes" })
-    public JsonNode postAddAdminToOrg( String organizationName, String email, String password, String token ) throws IOException {
-        JsonNode node = null;
-
-        MultivaluedMap formData = new MultivaluedMapImpl();
-        formData.add( "email", email );
-        formData.add( "password", password );
-
-        node = mapper.readTree( resource().path( "/management/organizations/" + organizationName + "/users" )
-                .queryParam( "access_token", token ).accept( MediaType.APPLICATION_JSON )
-                .type( MediaType.APPLICATION_FORM_URLENCODED ).post( String.class, formData ));
-
-        assertNotNull( node );
-        logNode( node );
-        return node;
-    }
-
-
     @Test
     public void putAddToOrganizationFail() throws Exception {
 
-        Map<String, String> originalProperties = getRemoteTestProperties();
+        Map<String, Object> originalProperties = getRemoteTestProperties();
 
         try {
-            setTestProperty( PROPERTIES_SYSADMIN_APPROVES_ADMIN_USERS, "false" );
-            setTestProperty( PROPERTIES_SYSADMIN_APPROVES_ORGANIZATIONS, "false" );
-            setTestProperty( PROPERTIES_ADMIN_USERS_REQUIRE_CONFIRMATION, "false" );
-            setTestProperty( PROPERTIES_SYSADMIN_EMAIL, "sysadmin-1@mockserver.com" );
-
-            String t = adminToken();
-            MultivaluedMap formData = new MultivaluedMapImpl();
-            formData.add( "foo", "bar" );
+            setTestProperty(PROPERTIES_SYSADMIN_APPROVES_ADMIN_USERS, "false");
+            setTestProperty(PROPERTIES_SYSADMIN_APPROVES_ORGANIZATIONS, "false");
+            setTestProperty(PROPERTIES_ADMIN_USERS_REQUIRE_CONFIRMATION, "false");
+            setTestProperty(PROPERTIES_SYSADMIN_EMAIL, "sysadmin-1@mockserver.com");
+
+            String t = this.getAdminToken().getAccessToken();
+            Form form = new Form();
+            form.add("foo", "bar");
             try {
-                resource().path( "/management/organizations/test-organization/users/test-admin-null@mockserver.com" )
-                        .queryParam( "access_token", t ).accept( MediaType.APPLICATION_JSON )
-                        .type( MediaType.APPLICATION_FORM_URLENCODED ).put( String.class, formData );
-            }
-            catch ( UniformInterfaceException e ) {
-                assertEquals( "Should receive a 400 Not Found", 400, e.getResponse().getStatus() );
+                this.org().getResource(false).path("/users/test-admin-null@mockserver.com")
```

Comment:
Was the test not valid against the management endpoint?  Just noticed the difference between the deleted test and the new test logic.